transform/models/staging/dof_mdsa/stg_footprints_block.sql

```diff
@@ -0,0 +1,13 @@
+with footprints_block as (
+    select 
+f."release",
+f."capture_dates_range",
+f."geometry",
+b."GEOID20",
+b."NAME20"
+from raw_dev.building_footprints.california_building_footprints f
```

Comment:
This and elsewhere should be a `{{source}}` call

Reply:
Will update